### PR TITLE
New graffiti.

### PR DESCRIPTION
**Pull request by xxalpha**
Adds 'cyka', pro lizard and anti lizard graffiti. (Lizard related graffiti are only available if the mutant races config is on)

graffiti
order of cykas: cyka1, cyka2, cyka3
Which cykas should be used?

### CODE REVIEW

**Comment by Nienhaus** [comment]
:+1:

**Comment by pudl** [comment]
cyka looks nice, the others are funny ideas but poorly sprited

**Comment by WJohn** [comment]
It's not like graffiti is supposed to be well sprited, but these seem fine to me :+1:

**Comment by RemieRichards** [comment]
The Lizard graffiti only appears when they're enabled? that's brilliant.

:+1:

**Comment by xxalpha** [comment]
I might keep only cyka1. :ru:

File: html/changelogs/xxalpha-newgraffiti.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+author: xxalpha
+# Optional: Remove this file after generating master changelog. Useful for PR changelogs that won't get used again.
+delete-after: True
+# Any changes you've made. See valid prefix list above.
+# INDENT WITH TWO SPACES. NOT TABS. SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading. Just remove the brackets when you add new shit.
+# Please surround your changes in single (') or double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes:
+-  rscadd: "Added new graffiti: cyka1, cyka2, cyka3, prolizard, antilizard"
```
Comment (Deantwo) [inline html/changelogs/xxalpha-newgraffiti.yml]: Space space dash space.

**Comment by Miauw62** [comment]
should probably be fukc instead of fuck, imo.

**Comment by kevinz000** [comment]
CHELP WHEN!?

**Comment by Deantwo** [comment]
I don't get why it says ```F*** Lizard ?```?

Also what is ```CYKA```?

**Comment by RemieRichards** [comment]
@Deantwo CYKA is pronounced a bit like "Sooka", and it's Russian for bitch
also Fuck lizard is just anti lizard propaganda

**Comment by Niknakflak** [comment]
I think they look bad and people will spam them to be assholes.  Graffiti really shoudnt have already spelled out words.  If you want words, you gotta do it the hard way. One letter at a time

**Comment by smet19** [comment]
Add хуй please

**Comment by Guap6512** [comment]
Yeah, we need Xyu and блядь

**Comment by paprka** [comment]
russians pls go

**Comment by LORDIKAMENCHIK** [comment]
Add "ХОС ХУЕСОС" plz.

**Comment by Vallat** [comment]
Add "PIZDEC" plz

**Comment by dannno** [comment]
ahahahahahahahhaha

**Comment by paprka** [comment]
holy shit

**Comment by Vallat** [comment]
"HOS HYESOS" is really nice idea, add it too. plz

**Comment by alexix1989** [comment]
Add "Петух" plz.

**Comment by dannno** [comment]
HAHAHAHAHA AHAHAHHAHA

**Comment by SconesC** [comment]
russians begone

**Comment by MyLittleLoly** [comment]
this post was captured by communism.

**Comment by Kraseo** [comment]
﻿ＥＮＧＡＧＩＮＧ  ＣＨＩＮＥＳＥ  ＩＮＶＡＤＥＲ

**Comment by pudl** [comment]
he kill my kite

**Pull request merged**